world/templates/index.html

```diff
@@ -29,16 +29,16 @@
 </head>
 <body>
 {% block header %}
-  <div class="row">
+  <div class="row top-header">
     <div class="title-header">
       <h2 class="project-title caps">{{ settings.site_title }}</h2>
       <h3 class="project-subtitle">{{ settings.site_description }}</h3>
     </div>
     <div class="information-instructions">
-      <p class="organization-information">{{ settings.about_text }} This site is designed to help people prepare for disasters that can occur in their area.</p>
+      <p class="organization-information">{{ settings.about_text }}</p>
       <p>{{ location.disaster_description }}</p>
-      <p class="location-instructions-large">Give your location by typing in the box below, clicking on the map, or clicking 'find me'. You'll get a personalized report on your risks and how to prepare.</p>
-      <p class="location-instructions-small">Give your location by typing in the box below or clicking 'find me'. You'll get a personalized report on your risks and how to prepare.</p>
+      <p class="location-instructions-large">Give your location by typing in the box below, clicking on the map, or clicking 'find me'. You'll get a personalized report on your risks and steps you can take to prepare.</p>
+      <p class="location-instructions-small">Give your location by typing in the box below or clicking 'find me'. You'll get a personalized report on your risks and steps you can take to prepare.</p>
```

Comment:
Does only one of these paragraphs show on any given client?  (If so, ignore this comment)